src/main/java/project_16x16/scene/AudioSettings.java

```diff
@@ -57,18 +57,13 @@ void mouseReleased(processing.event.MouseEvent e) {
 		if (apply.hover()) {
 			// TODO apply settings
 			Notifications.addNotification("Sound Settings Applied", "Your configuration has been successfully applied.");
-			return;
 		}
 	}
 	
 	@Override
 	void keyReleased(KeyEvent e) {
-		switch (e.getKeyCode()) {
-			case PConstants.ESC : // Pause
-				game.returnScene();
-				break;
-			default :
-				break;
+		if (e.getKeyCode() == PConstants.ESC) { // Pause
+			game.returnScene();
```

Comment:
I'd say put the switch case back to accommodate when we do add other key combinations.

Reply:
Good point. I didn't think about future key combinations when I made the change.